src/AutoROM.py

```diff
@@ -153,32 +164,32 @@ def torrent_tar_to_buffer():
     success = False
     while not success:
         if tries > 2:
-            raise RuntimeError("Failed to download ROMs from torrent, please try again or report this issue.")
-
-        try:
-            tries += 1
-            timeit = 0
-
-            # libtorrent params
-            ses = lt.session()
-            params = lt.parse_magnet_uri(uri)
-            params.save_path = save_path
-            handle = ses.add_torrent(params)
-
-            # download roms as long as state is not seeding
-            while handle.status().state != 5:
-                # some sleep helps
-                time.sleep(1)
-                timeit += 1
-
-                if timeit >= 20:
-                    raise TimeoutError("Took too long to download and reach seeding.")
-
-            success = True
-
-        except TimeoutError:
-            pass
-
+            raise RuntimeError("Tried to download ROMs 3 time, which have all failed, please try again or report this issue.")
+
+        tries += 1
+
+        # libtorrent params
+        ses = lt.session()
+        params = lt.parse_magnet_uri(uri)
+        params.save_path = save_path
+        handle = ses.add_torrent(params)
+
+        # download roms as long as state is not seeding
+        timeit = 1
+        while handle.status().state != 5:
```

Comment:
Do we want to check for either state 4 and 5 instead of only 5? Just a thought

Reply:
I was unsure, I had it in the original PR. I will add now and see what happens for users